x/emissions/keeper/keeper.go

```diff
@@ -1705,11 +1705,16 @@ func (k *Keeper) DripTopicFeeRevenue(ctx context.Context, topicId TopicId, block
 		return err
 	}
 
+	topic, err := k.GetTopic(ctx, topicId)
+	if err != nil {
+		return err
+	}
+
 	moduleParams, err := k.GetParams(ctx)
 	if err != nil {
 		return err
 	}
-	epsilon := moduleParams.Epsilon
+	epsilon := topic.Epsilon
```

Comment:
we shouldn't be using a topic level param to effect a global issue. this gives topics an incentive to minimize epsilon regardless of if the lesser scale is warranted (for maximally accurate inferences). => bad idea. should be new global parameter in this case